app/_locales/zh/messages.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
????

Reply:
So I think this may have been a configuration issue. This would be generic Chinese (?), for which we currently have no translations. We have translations for zh_CN (Mainland China Chinese) and zh_TW (Taiwan Chinese). Will look into why this file was created.

Reply:
Same here. I've reached out to Crowdin for clarity on what's happening here.

Reply:
Hi there! Could you please specify project name and share more details? It looks like the issue here is that folder "zh" was created, right?

Reply:
@Gudahtt, Andriy from Crowdin has modified configuration so that this file is no longer created, and the Chinese translations (simplified and traditional) are correctly recognized.